delivery_auto_refresh/__manifest__.py

```diff
@@ -4,13 +4,13 @@
 {
     "name": "Auto-refresh delivery",
     "summary": "Auto-refresh delivery price in sales orders",
-    "version": "16.0.1.0.1",
+    "version": "16.0.2.0.0",
     "category": "Delivery",
     "website": "https://github.com/OCA/delivery-carrier",
     "author": "Tecnativa, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "depends": ["delivery"],
+    "depends": ["delivery", "sale_order_carrier_auto_assign"],
```

Comment:
You can't add a new dependency in this module in stable, but also because those using this module since the beginning don't want to have a new one.